test/AutoDiff/differentiable_attr_parse.swift

```diff
@@ -57,6 +57,35 @@ public func squareRoot() -> Self {
   return lhs
 }
 
+func constFunc(_ slope: Float) -> ((Float) -> (Float, (Float) -> Float)) {
```

Comment:
Could you please remove functions without `@differentiable(linear)`?
* `constFunc`
* `const3`
* `const5`
* `const6`

`test/AutoDiff/differentiable_attr_parse.swift` only invokes the parser, so later passes (Sema, SIL) won't be run/tested.

Reply:
Ahh okay!